bin/package/installation/initd.sh

```diff
@@ -0,0 +1,220 @@
+### END INIT INFO
+
+# If you modify this, please make sure to also edit systemd.service
+
+OS_DIR_BIN="/usr/bin"
+OS_DIR_CONFIG="/etc/mysterium-node"
+OS_DIR_DATA="/var/lib/mysterium-node"
+OS_DIR_LOG="/var/log/mysterium-node"
+OS_DIR_RUN="/var/run/mysterium-node"
+
+# Process name (For display)
+DAEMON_NAME="mysterium-node"
+#Daemon name, where is the actual executable
+DAEMON_BIN="$OS_DIR_BIN/myst"
+# User and group
+DAEMON_USER="mysterium-node"
+DAEMON_GROUP="mysterium-node"
+# PID file for the daemon
+DAEMON_PIDFILE="$OS_DIR_RUN/daemon.pid"
+# Logging
+DAEMON_STDOUT="$OS_DIR_LOG/daemon.log"
+DAEMON_STDERR="$OS_DIR_LOG/error.log"
+# Command-line options that can be set in /etc/default/mysterium-node.
+# These will override any config file values.
+DAEMON_DEFAULT="/etc/default/mysterium-node"
+
+# Check for sudo or root privileges before continuing
+if [ "$UID" != "0" ]; then
+    echo "You must be root to run this script"
+    exit 1
+fi
+
+# If the daemon is not there, then exit.
+if [ ! -x $DAEMON_BIN ]; then
+    echo "Executable $DAEMON_BIN does not exist!"
+    exit 5
+fi
+
+# Create directory for pid file
+PIDDIR=`dirname $DAEMON_PIDFILE`
+if [ ! -d "$PIDDIR" ]; then
+    mkdir -p $PIDDIR
+    chown $DAEMON_USER:$DAEMON_GROUP $PIDDIR
+fi
+
+
+# Create directory for logs
+LOGDIR=`dirname $DAEMON_STDOUT`
+if [ ! -d "$LOGDIR" ]; then
+    mkdir -p $LOGDIR
+    chown -R -L $DAEMON_USER:$DAEMON_GROUP $LOGDIR
+fi
+LOGDIR=`dirname $DAEMON_STDERR`
+if [ ! -d "$LOGDIR" ]; then
+    mkdir -p $LOGDIR
+    chown -R -L $DAEMON_USER:$DAEMON_GROUP $LOGDIR
+fi
+
+if [ -r /lib/lsb/init-functions ]; then
+    source /lib/lsb/init-functions
+fi
+
+# Override init script variables with DEFAULT values
+if [ -r $DAEMON_DEFAULT ]; then
+    source $DAEMON_DEFAULT
+fi
+
+function log_failure_msg() {
+    echo "$@" "[ FAILED ]"
+}
+
+function log_success_msg() {
+    echo "$@" "[ OK ]"
+}
+
+function start() {
+    # Check that the PID file exists, and check the actual status of process
+    if [ -f $DAEMON_PIDFILE ]; then
+        PID="$(cat $DAEMON_PIDFILE)"
+        if kill -0 "$PID" &>/dev/null; then
+            # Process is already up
+            log_success_msg "$DAEMON_NAME process is already running"
+            return 0
+        fi
+    else
+        su -s /bin/sh -c "touch $DAEMON_PIDFILE" $DAEMON_USER &>/dev/null
+        if [ $? -ne 0 ]; then
+            log_failure_msg "$DAEMON_PIDFILE not writable, check permissions"
+            exit 5
+        fi
+    fi
+
+    # Launch process
+    echo "Starting $DAEMON_NAME..."
+    start-stop-daemon \
+        --start \
+        --quiet \
+        --background \
+        --no-close \
+        --make-pidfile \
+        --pidfile $DAEMON_PIDFILE \
+        --user $DAEMON_USER \
+        --group $DAEMON_GROUP \
+        --exec $DAEMON_BIN \
+        -- \
+        --config-dir=$OS_DIR_CONFIG \
+        --data-dir=$OS_DIR_DATA \
+        --runtime-dir=$OS_DIR_RUN \
+        service \
```

Comment:
Currently we have travis job `name: "Ubuntu docker image for node"` to test .deb installation on Ubuntu.
But it did not identified this error. 
Do You have ideas how to do E2E test of Debian service?

Reply:
This job just builds a docker image based on Ubuntu. It installs dep package, but in this case installation of dep package was not broken, that is why we didn't find this issue. 
Docker uses it's own `docker-entrypoint.sh` for starting services. But broken was system service.
Not sure how to test it correctly with a systemd service.

Reply:
Created issue for the future improvement: #417